go/vt/vttablet/tabletmanager/vreplication/controller_plan_test.go

```diff
@@ -108,36 +141,40 @@ func TestControllerPlan(t *testing.T) {
 	}, {
 		in:  "update _vt.vreplication set state='Running', id = 2 where id = 1",
 		err: "id cannot be changed: id = 2",
-	}, {
-		in:  "update _vt.vreplication set state='Running'",
-		err: "invalid where clause:",
-	}, {
-		in:  "update _vt.vreplication set state='Running' where a = 1 and id = 2",
-		err: "invalid where clause: where a = 1 and id = 2",
-	}, {
-		in:  "update _vt.vreplication set state='Running' where a = 1",
-		err: "invalid where clause: where a = 1",
-	}, {
-		in:  "update _vt.vreplication set state='Running' where id > 1",
```

Comment:
I assume this is now a valid query. Why not keep this test?

Reply:
The new tests with conditions like `where a = 1` replace these. The change is that we now accept any where clause, and don't really look at the contents.